src/frontend/apps/standalone_site/src/features/Contents/features/ClassRoom/components/Read/ClassRoomItem.tsx

```diff
@@ -45,33 +94,38 @@ const ClassRoom = ({ classroom }: { classroom: ClassroomLite }) => {
         }
         footer={
           <Fragment>
-            {classroom.starting_at && (
-              <Box gap="small" align="center" direction="row">
-                <Box>
-                  <FormSchedule size="medium" color="blue-active" />
+            <Box gap="small" align="center" direction="row">
```

Comment:
If you add a `starting_date` and a `duration`, the row are now on the same line, I guess you can remove this Box.